_posts/2024-02-14-Are-You-Ready-to-Low-Code-Green?-Introduction to-Sustainable-Canvas-App-and-Power-Automate-Development

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Are You Ready to Low-Code Green? Introduction to Sustainable Canvas App and Power Automate Development
+author: milla.vaananen
+excerpt: >
+   Some insight and ideas how to implement green coding practicisies in Canvas App and Power Automate Development.
+tags:
+ - Green Coding
+ - Canvas App
+ - Power Platform
+ - Power Automate
+---
+
+![Illustartion of how green software divedes to energt effiency, hardware effiency and carbon awarness](/img/2024-Power-Platform/Green%20Sofware%20Principles.png)
+> Picture: [Green Software Foundation](https://learn.greensoftware.foundation/introduction)
+
+The ICT sector (including data centers, communication networks, and user devices) is estimated to account for [around 4-6% of global electricity usage.](https://post.parliament.uk/research-briefings/post-pn-0677/) When discussing CO2 or greenhouse gas emissions, the ICT sector's emissions primarily stem from electricity consumption. As an app developer, I don’t have the power to dictate the type of electricity end-users or data centers use; therefore, I should focus on reducing the energy consumption of the app.
+
+When working with the Power Platform, one might easily think that developers can't do much. The platform takes care of most things like scaling, data savings, and cloud servers. Here, green coding comes into play.
```

Comment:
could make "Power Platform" a link to it, I'm unfamiliar with it

Reply:
added!